backstage/values.yaml

```diff
@@ -81,6 +81,18 @@ externalSecrets:
       key: AUTH_GOOGLE_CLIENT_ID
     - name: google-client-secret
       key: AUTH_GOOGLE_CLIENT_SECRET
+    - name: circleci_auth_token
```

Comment:
This should be simplified after we can start using https://github.com/external-secrets/kubernetes-external-secrets/pull/238